### DISCUSSION TITLE
【求助】有个人在不停地肘击飞行载具的大门！！！我该怎么办？？？

### DISCUSSION THREAD

**Discussion by LBK-miq** [share121/inter-knot#2825] [其他]
![QQ截图20240818215025](https://github.com/user-attachments/assets/d5e06e1a-b966-4eb9-a6f0-2d24a0853201)

如题，本人在空洞里面搭乘飞行载具执行任务，正飞得好好的呢，忽然有个黑人穿着黄紫相间的球衣就冲上来了，疯狂地用手肘击打飞行载具的大门，嘴里还念念有词什么“what can I say”之类的话，给一整个飞行载具上的人都吓坏了！！！我寻思起飞的时候也检查过了，没发现有什么以骸闯入的迹象啊？？？只好紧急求助各位绳网大佬，这下该怎么办？？？

**我还在天上呢！！！我可不想坠机！！！** <img src="https://img-static.mihoyo.com/communityweb/upload/9939adf214795d97dad9abd90265aab2.png" alt="帕姆_哭哭" width="75" >

**Comment by WXZ1025** [comment]
跟他说：
你说你↗️不想↘️在这里☹️
我也不想在这里😖
但天黑的太快😩
想走早就来不及↗️
我↗️爱↘️你😘
可惜关系变成没关系💔
问题是没问题👌
于是我们继续↘️

得↗️得↘️得↗️得↘️得↗️得↘️得↗️得↘️得↗️得↘️得↗️得↘️得↗️得↘️得↗️得↘️得↗️得↘️得↗️得↘️得↗️得↘️得↗️得↘️（吉他音）

拿着笔想写点东西🤔
以为是武器能伸张正义！
没人理也没关系🥺
至少我还有你🙂
你说你不想在这里🙁
我也不想在这里😖
但天黑的太快🌚
想走早就来不及[捂脸]
哦我爱你😘
但是关系变成没关系💔
问题是没问题👌
于是我们继续🤕

得↗️得↘️得↗️得↘️得↗️得↘️得↗️得↘️得↗️得↘️得↗️得↘️得↗️得↘️得↗️得↘️得↗️得↘️得↗️得↘️得↗️得↘️得↗️得↘️（吉他音）

哦我爱你😘
那些问题我都没关系😌
但是离开了这里【流泪】
于是我们不再年轻👋

**Comment by Manbow1997** [comment]
对他使用“闪电旋风劈”吧！

**Comment by yangtiansa** [comment]
man ！

**Comment by GrunderIndustries** [comment]
最佳答案是下单我司的MQ-99无人机 然后用那门强悍的20mm机炮将其打个尸骨无存
